tests/integration/controller/withwatcher/suite_with_watcher_test.go

```diff
@@ -178,21 +178,34 @@ var _ = BeforeSuite(func() {
 	}
 
 	remoteClientCache = remote.NewClientCache()
-	skrChartCfg := &watcher.SkrWebhookManagerConfig{
+	skrChartCfg := watcher.SkrWebhookManagerConfig{
 		SKRWatcherPath:         skrWatcherPath,
 		SkrWebhookMemoryLimits: "200Mi",
 		SkrWebhookCPULimits:    "1",
-		IstioNamespace:         istioSystemNs,
-		IstioGatewayName:       gatewayName,
-		IstioGatewayNamespace:  kcpSystemNs,
 		RemoteSyncNamespace:    controller.DefaultRemoteSyncNamespace,
-		CACertificateName:      caCertificateName,
 	}
 
-	caCertCache := watcher.NewCertificateCache(5 * time.Minute)
+	certificateConfig := watcher.CertificateConfig{
+		IstioNamespace:      istioSystemNs,
+		RemoteSyncNamespace: controller.DefaultRemoteSyncNamespace,
+		CACertificateName:   caCertificateName,
+		AdditionalDNSNames:  nil,
```

Comment:
`AdditionalDNSNames` is nil here and initialized with empty list here tests/integration/watcher/certificate_test.go, can you align this? Maybe just not set it.